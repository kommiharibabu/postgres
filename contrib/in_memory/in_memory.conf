shared_preload_libraries = 'in_memory'
