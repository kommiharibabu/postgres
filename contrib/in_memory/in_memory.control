# in_memory extension
comment = 'in memory table support'
default_version = '1.0'
module_pathname = '$libdir/in_memory'
relocatable = true
