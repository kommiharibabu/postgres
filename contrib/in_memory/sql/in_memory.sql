CREATE EXTENSION in_memory;

DROP EXTENSION pg_stat_statements;
