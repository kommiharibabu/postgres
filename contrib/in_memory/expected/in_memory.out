CREATE EXTENSION pg_stat_statements;
DROP EXTENSION pg_stat_statements;
